client/src/components/DropdownMenu.tsx

```diff
@@ -58,7 +87,9 @@ export class DropdownMenu extends React.Component {
 
     const aria_label = dropdown_a11y_txt
       ? dropdown_a11y_txt
-      : _.isString(dropdown_trigger_txt) && dropdown_trigger_txt;
+      : _.isString(dropdown_trigger_txt)
+      ? dropdown_trigger_txt
+      : undefined;
```

Comment:
super nitpick: can we just use a `let` here? nested ternaries hurt my brain

Reply:
How about an `if` statement inside an immediately invoked function (or a  `get_aria_label` declared elsewhere or something)?

Reply:
yup, I already did iif in my commit! https://github.com/TBS-EACPD/infobase/pull/1056/commits/db235bd5e3966c504c0db8e65cd01f6435599154